packages/core/src/promotionEvaluations/client/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * PromotionEvaluations clients.
+ *
+ * @module promotionEvaluations/client
+ * @category PromotionEvaluations
+ * @subcategory Clients
+ */
+import { warnDeprecatedMethod } from '../../helpers';
+
+warnDeprecatedMethod(
+  '@fps/core',
+  '@fps/core/promotionEvaluations/client',
+  '@fps/core/promotionEvaluations',
```

Comment:
I think you mean '@farfetch/blackout-core' :)

Reply:
Oops! I don't know how that got there. Fixed 🙂